src/teaching-element/Assessment/SingleChoice.vue

```diff
@@ -43,7 +44,7 @@ export default {
     retake: { type: Boolean, default: false },
     submission: { type: Number, default: null }
   },
-  data: vm => ({ userAnswer: vm.submission || null }),
+  data: vm => ({ userAnswer: isNull(vm.submission) ? null : vm.submission }),
```

Comment:
Any reason why it must be this way?

Isn't this sufficient?
```suggestion
  data: vm => ({ userAnswer:  vm.submission }),
```

Reply:
😅 whoops.